### PR TITLE
add contact form for self-hosted community license.

### LINKED ISSUE #2192: Add contact form for self-hosted community license
- [Internal discussion](https://gitpod.slack.com/archives/C01LDA28WV6/p1653035219696759)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #2192 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2237"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2237"></a>

### CODE REVIEW

**Comment by chrifro** [comment]
Thanks @nisarhassan12 

Some comments:

- [ ] remove the country field from the form as Philipp suggested

- [ ] show automatically to the top of the confirmation page after sending the form. Currently I need to scroll up to see it
Bildschirmfoto 2022-06-14 um 14 27 41

- [ ] Can we add "others" to the infrastructure options or alternatively have it as an optional field? cc @phimae 

Otherwise it LGTM ✨

Where is the email currently sent to?

**Comment by nisarhassan12** [comment]
@chrifro The email is currently sent to `sales@gitpod.io`

**Review by chrifro — APPROVED** [review]
LGTM

Let's hand it over to @lucasvaltl and @phimae to follow up on the email logic and to hit the merge button. 🙌

**Review by lucasvaltl — CHANGES_REQUESTED** [review]
Just have a small wording change that I think would make it more clear what happens next and also what to do if you would like to install in an air gapped environment.

File: src/routes/community-license.svelte [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    description: "Request a free community license",
+    title: "Community License",
+    norobots: true,
+  }}
+/>
+
+<Header tight={true}>
+  <div slot="content" bind:this={sectionStart}>
+    <h1 class="h2">Request free community license</h1>
+    <p>Fill out the form and receive the free self-hosted community license.</p>
```
[the remark this answers is not in the document]
Comment (phimae) [reply src/routes/community-license.svelte:147]: That makes sense @lucasvaltl. To reduce salesyness and shorten we could do this wording:

H1: Receive the free community license via email
H2: Fill out the form and immediately receive the free self-hosted community license via email. If you require support for air gapped environments, please [request an enterprise license](https://www.gitpod.io/enterprise-license) to get a free trial. (like Lucas suggested)

[the remark this answers is not in the document]
Comment (nisarhassan12) [reply src/routes/community-license.svelte]: I have updated the text.

**Comment by phimae** [comment]
@nisarhassan12 thanks for the quick PR! A few comments: 

- [ ] can you make it so that when the form is submitted, it creates a row with the response in [this sheet](https://docs.google.com/spreadsheets/d/1FDeP3Y1Fg__SqDOUzedmC4SzM1qg0o7YLuTI3kkaHrc/edit#gid=0)? The email can still be sent to the alias like it does atm.
- [ ] copying over from Lucas's thread: can you change the wording in the header and sub header to:
- H1: Receive the free community license via email
- H2: Fill out the form and immediately receive the free self-hosted community license via email. If you require support for air gapped environments, please [request an enterprise license](https://www.gitpod.io/enterprise-license) to get a free trial.

- Regarding Christin's suggestion for adding other infradtructures: @lucasvaltl do you think that is helpful given that we currently don't support others? Maybe to gauge appetite?

File: src/routes/community-license.svelte [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+<Header tight={true}>
+  <div slot="content" bind:this={sectionStart}>
+    <h1 class="h2">Receive the free community license via email</h1>
+    <p>
+      Fill out the form and immediately receive the free self-hosted community
+      license via email. If you require support for air gapped environments,
+      please request an <a href="/enterprise-license">enterprise license</a> to
+      get a free trial. Fill out the form and immediately receive the free
+      self-hosted community license via email. If you want to try out Gitpod in
+      an air gapped environment, please request an
```
Comment (phimae) [inline src/routes/community-license.svelte:153]: @nisarhassan12 this is doubled - can you pls remove the second part here?

File: src/routes/community-license.svelte [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+          rows="4"
+          bind:value={formData.message.value}
+          bind:element={formData.message.el}
+          name="message"
+          id="message"
+        />
+        <div class="my-4">
+          <Checkbox
+            hasError={isFormDirty && !formData.country.valid}
+            label="I consent to having this website store my submitted information so that the sales team can respond to my inquiry."
```
Comment (phimae) [inline src/routes/community-license.svelte:285]: can you pls replace "the sales team" with "the Gitpod team"?

**Review by phimae — APPROVED** [review]

**Pull request merged**